etc/dbus-serialbattery/dbus-serialbattery.py

```diff
@@ -46,6 +47,9 @@
     {"bms": Renogy, "baud": 9600, "address": b"\x30"},
     {"bms": Renogy, "baud": 9600, "address": b"\xF7"},
     {"bms": Seplos, "baud": 19200},
+    # {"bms": HeltecModbus, "baud": 9600, "address": 3},
+    # {"bms": HeltecModbus, "baud": 9600, "address": 2},
+    {"bms": HeltecModbus, "baud": 9600, "address": 1},
```

Comment:
Please sort both alphabetically. You can remove the commented lines, since you can set the address in config file.